Marlin/Configuration.h

```diff
@@ -897,7 +897,7 @@
 #if ANY(PIDTEMP, PIDTEMPBED, PIDTEMPCHAMBER)
   //#define PID_OPENLOOP          // Puts PID in open loop. M104/M140 sets the output power from 0 to PID_MAX
   //#define SLOW_PWM_HEATERS      // PWM with very low frequency (roughly 0.125Hz=8s) and minimum state time of approximately 1s useful for heaters driven by a relay
-  #define PID_FUNCTIONAL_RANGE 10 // If the temperature difference between the target temperature and the actual temperature
+  #define PID_FUNCTIONAL_RANGE 20 // If the temperature difference between the target temperature and the actual temperature
```

Comment:
Smaller values are preferred, of course, implying a better tuned PID. But presumably this also impacts tuning (?) so smaller values might result in some poor tuning when the starting PID is very bad.

IOW it seems like a good idea to raise this value, just a question whether to use 20 or something else.